Libbna/CUSTOM-CMS
# #169: Added Comment Feature

### Work In Progress 

**Testing Steps:**

- Go to any article page.
- Scroll down to the bottom.
- check if the comment box appears or not.
- try commenting.
- Refer to the following screenshot.

![image](https://user-images.githubusercontent.com/81369580/150973645-291cf945-f83c-4383-91b3-09a4ae289ce6.png)
